### PR TITLE
[blog] Free friday - First draft, preview

### PR DESCRIPTION
**Pull request by borsi**
~~I am not a smart man. I managed to branch from the previous post, so this now looks like a cumulative pull request with the previous post. :\~~ NOT ANYMORE! BOW BEFORE ME FOR I AM **MASTER OF GIT**. HAHA.

Anyway. Pls review. 💃

### CODE REVIEW

File: _drafts/2017-06-20-free-friday.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title:  "Free friday for all - Chapters in Sense/Net"
+author: borsi
+tags: [community, developers, communication, company culture, freefriday]
+---
+In their [2004 IPO letter][64bbc0a2], founders Larry Page and Sergey Brin highlighted their idea of the famous "Google 20%". A lot of time has passed since then and the legend faded too: is it really 20% of their time, or is it more like "Google 120%"?
+
+  [64bbc0a2]: https://abc.xyz/investor/founders-letters/2004/ipo-letter.html "Google IPO letter"
+
+Nontheless, we wanted to empower our people so we decided to give it - or at least **a** version of it - a go.
```
Comment (iviczl) [inline _drafts/2017-06-20-free-friday.md]: Nonetheless

File: _drafts/2017-06-20-free-friday.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+---
+
+> We encourage our employees, in addition to their regular projects, to spend 20% of their time working on what they think will most benefit Google. This empowers them to be more creative and innovative. Many of our significant advances have happened in this manner. For example, AdSense for content and Google News were both prototyped in “20% time.” Most risky projects fizzle, often teaching us something. Others succeed and become attractive businesses.
+
+They wrote. It almost sounds too good to be true: our people do what they think is best for us, and it truly gets better. It is always important to remind ourselves before getting overly excited about things like this, that _[we are not Google][bfca94a4]_. So before rushing to the solution, we need to think about our on corporate structure, and problems we need solving.
```
Comment (iviczl) [inline _drafts/2017-06-20-free-friday.md]: ... we need to be solved.

File: _drafts/2017-06-20-free-friday.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+  [bfca94a4]: https://blog.bradfieldcs.com/you-are-not-google-84912cf44afb "You are not google - Ozan Onay"
+
+### How Sense/Net Inc. works
+
+Sense/Net Inc. - the company behind sensenet ECM - has two products. One actual product (sensenet ECM) and a solution integrator service: since we know our product the best, we can create unique solutions for companies all around the world (we also do training and quite a lot of support, but let's just consider these parts of one or both of the products I just mentioned).
+
+This results in teams that not just work on different projects, they do **different kinds of things**. In time, people end up with vastly different know-hows and knowledge levels. It is hard to admit, but _sometimes_ there are multiple solutions for the same problem. Or worse, there are the same solutions implemented several times for the same problem.
```
Comment (iviczl) [inline _drafts/2017-06-20-free-friday.md]: This results in that the teams not just...

**Comment by balazskeszei** [comment]
I don't see the end part of this post. We need "conclusion" or next steps or expectations. Now it seems that you finish it without having a frame of it.
Like, we should mention the framework of the idea, how we manage it internally, what do we expect from the participants (blog/presentation/etc).

File: _drafts/2017-06-20-free-friday.md (hunk 3 above)
Comment (borsi) [reply _drafts/2017-06-20-free-friday.md]: This has a different meaning. I want to say that the teams (that do things differently) are the result of the causes.

**Comment by borsi** [comment]
Added an ending and fixed some grammar. :) I also added a "cover" image.

**Comment by herflis** [comment]
Pls move the images to the /img/posts/ folder! :)

**Comment by borsi** [comment]
Done :)

**Review by gallayl — APPROVED** [review]

**Review by herflis — APPROVED** [review]
Maybe 'Fedex Day' at the end could be a link to a definition what Fedex Day is, great job anyway

**Comment by borsi** [comment]
Thanks, will do :) and maybe I'll change the filename to an appropriate publish date. What do you think, 06.27 or 06.28 would be better?

**Comment by balazskeszei** [comment]
June 28

**Pull request merged**